source/websocket_bootstrap.c

```diff
@@ -151,6 +152,7 @@ int aws_websocket_client_connect(const struct aws_websocket_client_connection_op
 
     ws_bootstrap->alloc = options->allocator;
     ws_bootstrap->initial_window_size = options->initial_window_size;
+    ws_bootstrap->manual_window_update = options->manual_window_management;
```

Comment:
name it "manual_window_management" all the way through

The underlying functions are called "increment read window" everywhere but HTTP, because HTTP/2 already had the term "WINDOW_UPDATE"